demo-magic.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
did you mean to push these two?

Reply:
oops, good eye. will rm!

Reply:
though I will say that is a cool bash script 😄

Reply:
thanks, I didn't write it